spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
-1 and I'd even go as far as explicitly add `@Qualifier("longRunningModuleDeployer")` (_i.e._ not rely on default qualifying behavior by name), as this is really a core feature of our approach that we have two "roles".